packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -181,7 +181,7 @@ export class MockRuntime
 			this.processBlobsP.reject(new Error("fake error"));
 		}
 		this.processBlobsP = new Deferred<void>();
-		await Promise.all(blobPs);
+		await Promise.all(blobPs).catch(() => {});
```

Comment:
i'm not sure this is correct, i think promise.all will exit waiting after the first reject. you want to want for all promise to resolve, regardless of resolve or reject.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all